nixos/tests/dsl.nix

```diff
@@ -119,6 +130,18 @@ in
         def path_to_name(path: str) -> str:
           return re.sub(r"^[ ]*${builtins.storeDir}/[a-z0-9]*-([a-zA-Z0-9_-]+)(\.drv)?", r"\1", path)
 
+        def parse_evanix_dry_run(output):
+          to_build = [ ]
+
+          for line in output.split("\n"):
+            if not re.match("nix-build --out-link .*$", line):
```

Comment:
You know, if `evanix` produced a json (what is built, what is left out, what are the costs and values) you wouldn't have to write regular expressions here:)